app/models/cdo/edit/change-status.js

```diff
@@ -5,6 +5,7 @@ function ViewModel (dog, backNav, errors) {
   this.model = {
     backLink: backNav.backLink,
     srcHashParam: backNav.srcHashParam,
```

Comment:
srcHashValue isn't needed (I don't think) as srcHashParam is the same but with '?src=' prefixed

Reply:
was there for another reason (input hidden), but had removed that anyway